lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -48,6 +48,12 @@ def setup_rabbitmq_connection_config
         # Time in seconds to wait before retrying a connection
         config :connect_retry_interval, :validate => :number, :default => 1
 
+        # The default connection timeout; zero means wait indefinitely
+        config :connection_timeout, :validate => :number
+
+        # Heartbeat delay in seconds. Default is 0 (no heartbeats)
```

Comment:
remove the comment about the default number. Maybe replace with "If not specified, no heartbeats are sent"